lib/gears/table.lua

```diff
@@ -88,9 +88,10 @@ end
 --- Check if a table has an item and return its key.
 -- @class function
 -- @name hasitem
--- @param t The table.
+-- @tparam table t The table.
 -- @param item The item to look for in values of the table.
--- @return The key were the item is found, or nil if not found.
+-- @treturn[1] string|number The key of the item.
```

Comment:
according you your other PR numeric indexes shouldn't be used with `@treturn`? https://github.com/awesomeWM/awesome/pull/2537/files

Reply:
When used they refer to different return values / possibilities, i.e. here it will either return the key or nil.

Reply:
i see, thanks for explaining this